app/models/mixins/relationship_mixin.rb

```diff
@@ -210,12 +240,21 @@ def ancestor_rels(*args)
   # Returns a list of ancestor records, starting with the root record and ending
   #   with the parent record
   def ancestors(*args)
+    return call_ancestry_method(:ancestors, args) if use_ancestry?
+
     Relationship.resources(ancestor_rels(*args))
   end
 
   # Returns a list of ancestor class/id pairs, starting with the root class/id
   #   and ending with the parent class/id
   def ancestor_ids(*args)
+    if use_ancestry?
+      anc_ids = read_attribute(self.ancestry_base_class.ancestry_column)
```

Comment:
can you use `send` to call the private parse method?

I've tried really hard to let us change the format of the ancestry column.